### PR TITLE
Add X2Go SHA1 fingerprints

### PR DESCRIPTION
**Pull request by hajgato**
I getting the following error:
```
./intro-HPC.tex:76: Extra \else.
\@include ...ediate \closeout \@partaux \fi \else
                                                  \deadcycles \z@ \@nameuse ...
l.76 \include{ch_troubleshooting}
```

### CODE REVIEW

File: intro-HPC/ch_troubleshooting.tex [hunk 1]
```diff
@@ -270,6 +270,17 @@ \section{Security warning about invalid host key}
...
 \includegraphics[width=0.5\textwidth]{putty_security_alert}
 
+\ifugent
+If you use X2Go client, you might get one of the following fingerprints:
+
+\xtwogoshaone
+
+\strong{If you get a message ``Host key for server changed'', do not click ``No'' until you verified the fingerprint.}.
+
+If it the fingerprint matches, click ``No'', and in the next pop-up screen (``if you accept the new host key...'', press ``Yes''.
```
Comment (stdweird) [inline intro-HPC/ch_troubleshooting.tex:280]: missing `)` at end of line

File: intro-HPC/ch_troubleshooting.tex [hunk 2]
```diff
@@ -304,10 +315,24 @@ \section{Warning message when first connecting to new host}
...
+\fi
+
 If it does, type  \strong{\emph{yes}}. If it doesn't, please contact support: \hpcinfo.
 
 \else
 \firsttimeconnection
+If you use X2Go, then you might get another fingerprint, then make sure that the fingerprint
+is displayed is one of the following ones:
+
+\xtwogoshaone
```
Comment (stdweird) [inline intro-HPC/ch_troubleshooting.tex:334]: this needs a `ifgent` block, no?

File: intro-HPC/ch_troubleshooting.tex [hunk 3]
```diff
@@ -270,6 +270,17 @@ \section{Security warning about invalid host key}
 
 \includegraphics[width=0.5\textwidth]{putty_security_alert}
 
+\ifugent
```
Comment (stdweird) [inline intro-HPC/ch_troubleshooting.tex:273]: ifgent?

**Comment by boegel** [comment]
@hajgato Is this still work-in-progress, or is it ready to be merged?

**Comment by hajgato** [comment]
@boegel This can be merged

**Pull request merged**